### PR TITLE
Ha/restore redacted files

### PR DESCRIPTION
**Pull request by hacodeorg**
## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: i18n/locales/vi-VN/dashboard/scripts.yml [hunk 1]
```diff
@@ -4267,7 +4266,10 @@ vi:
...
-              description_teacher: "Năm bài học tiếp theo tạo cơ hội cho học sinh sử dụng các kỹ năng viết mã của mình trong một dự án thực tiễn. Dự án này sẽ giúp các cá nhân tích lũy kinh nghiệm về lập trình và tạo ra sản phẩm mẫu để chia sẻ với đồng nghiệp và những người thân yêu. Theo dự kiến, đây sẽ là dự án bao gồm nhiều bài học hoặc nhiều tuần, trong đó học sinh sẽ dành thời gian khám phá, brainstorm, tìm hiểu về quy trình thiết kế, xây dựng và trình bày tác phẩm cuối cùng của mình.\n\nTrong giai đoạn khám phá, học sinh sẽ có thể xem ví dụ về các dự án trong cả Artist và Play Lab để lấy cảm hứng. Tiếp theo, học sinh sẽ tìm hiểu về quy trình thiết kế và cách thực hiện trong các dự án của chính mình. Sau đó, họ sẽ có không gian để tự tạo dự án của riêng mình trong Artist hoặc Play Lab (đây có thể là giai đoạn dài nhất của dự án). Sau đó, học sinh sẽ chỉnh sửa dự án của mình sau khi thử nghiệm và suy xét thêm. Cuối cùng, học sinh sẽ có thể trình bày tác phẩm đã hoàn thành của mình với bạn bè."
+              description_teacher: |-
```
Comment (hacodeorg) [inline i18n/locales/vi-VN/dashboard/scripts.yml:4269]: Is this okay?

File: i18n/locales/vi-VN/dashboard/scripts.yml [hunk 2]
```diff
@@ -5257,7 +5259,7 @@ vi:
               display_name: Nội dung khóa học bổ sung
           lessons:
             courseB_unplugged_MRF_2018:
-              name: courseB_unplugged_MRF_2018
+              name: courseB_unplugged_MRF\_2018
```
Comment (hacodeorg) [inline i18n/locales/vi-VN/dashboard/scripts.yml:5262]: Why does `_` become `\_`?

File: i18n/locales/vi-VN/dashboard/scripts.yml [hunk 3]
```diff
@@ -10233,7 +10314,7 @@ vi:
               description_teacher: |2-
                  [![common sense education](https://curriculum.code.org/static/img/common_sense_education.png)](https://creativecommons.org/)
 
-                Bài học này ban đầu được tạo bởi Common Sense Education. [Tìm hiểu thêm.] (https://www.commonsense.org/education/digital-citizenship/curriculum)
+                Bài học này ban đầu được tạo bởi Common Sense Education. [Tìm hiểu thêm.] \(<https://www.commonsense.org/education/digital-citizenship/curriculum>)
```
Comment (hacodeorg) [inline i18n/locales/vi-VN/dashboard/scripts.yml:10317]: Looks like a translation error (space between [] and ()). Is escaping `(` and `)` correct?

File: i18n/locales/sr-SP/dashboard/scripts.yml [hunk 4]
```diff
@@ -501,12 +502,6 @@ sr:
           description: 'Phase 2 Online: Blended Summer Study'
           description_short: 'Phase 2 Online: Blended Summer Study'
           lessons:
-            0: |-
```
Comment (hacodeorg) [inline i18n/locales/sr-SP/dashboard/scripts.yml:504]: Why were these English strings here?

File: dashboard/config/locales/authored_hints.pl-PL.json [hunk 5]
```diff
@@ -8831,18 +8831,18 @@
...
+          "hint1": "Aby nadać maski potworom, użyj bloku „zmiana kostiumu” ze zdarzeniem „po kliknięciu”:\n\n<xml><block type=\"gamelab_spriteClicked\"><title name=\"CONDITION\">\"when\"</title><value name=\"SPRITE\"><block type=\"gamelab_allSpritesWithAnimation\"><title name=\"ANIMATION\">\"healthy\"</title></block></value><next><block type=\"gamelab_setAnimation\"><title name=\"ANIMATION\">\"healthy\\_mask\"</title></block></next></block></xml>\n\nPamiętaj, aby przeciągnąć również mały czerwony blok z kliknięciem!",
```
Comment (hacodeorg) [inline dashboard/config/locales/authored_hints.pl-PL.json:8834]: This change modifies redacted content. Should investigate!

**Comment by deploy-code-org** [comment]
This comment is created by running: `bin/i18n/analyze_pr.rb -p 43795 -c`

Top 5/1244 files sorted by the number of changes:
- `dashboard/config/locales/dsls.es-MX.yml`, 16770 changes (8628 additions, 8142 deletions).
- `dashboard/config/locales/dsls.es-ES.yml`, 15718 changes (7790 additions, 7928 deletions).
- `dashboard/config/locales/dsls.de-DE.yml`, 14680 changes (7310 additions, 7370 deletions).
- `dashboard/config/locales/dsls.hi-IN.yml`, 7802 changes (3892 additions, 3910 deletions).
- `dashboard/config/locales/dsls.az-AZ.yml`, 7430 changes (3714 additions, 3716 deletions).

Top 5/5 commits sorted by the number of changes:
- Commit: Redistribute dashboard translations
  - 502862 changes (132450 additions, 370412 deletions).
  - 300 files changed.
  - File with the most changes: `dashboard/config/locales/dsls.es-MX.yml`, 16702 changes (8280 additions, 8422 deletions).
- Commit: Rerun restoration of redacted files
  - 42753 changes (17817 additions, 24936 deletions).
  - 114 files changed.
  - File with the most changes: `i18n/locales/in-TL/dashboard/scripts.yml`, 8885 changes (2793 additions, 6092 deletions).
- Commit: Redistribute course content translations
  - 23481 changes (20503 additions, 2978 deletions).
  - 292 files changed.
  - File with the most changes: `dashboard/config/locales/dsls.ru-RU.yml`, 2135 changes (2135 additions, 0 deletions).
- Commit: Redistribute apps/lib/blockly translations
  - 9552 changes (4503 additions, 5049 deletions).
  - 45 files changed.
  - File with the most changes: `apps/lib/blockly/in_tl.js`, 788 changes (394 additions, 394 deletions).
- Commit: Redistributing apps/i18n translations
  - 6991 changes (6595 additions, 396 deletions).
  - 105 files changed.
  - File with the most changes: `apps/i18n/spritelab/sk_sk.json`, 238 changes (122 additions, 116 deletions).

**Pull request closed without merging**